example/reader_util.h

```diff
@@ -336,4 +336,15 @@ float ndpi_flow_get_byte_count_entropy(const uint32_t byte_count[256], unsigned
 
 extern int nDPI_LogLevel;
 
+/* TODO: conditional compilation? */
+#if 1
+ #define LOG(log_level, args...)	\
```

Comment:
Why didn't you use `NDPI_LOG` ? It is already there.

Reply:
I thought I explained it in the first commit message, but probably I didn't make myself clear. Let me rephrase it.
From 66b759ca6, the ndpi_detection_module_struct object is an opaque data type for the user so you can't dereference it in reader_utils.c code; if you try, you get a compiler error, something similar to

../src/include/ndpi_define.h:188:26: error: invalid use of undefined type ‘struct ndpi_detection_module_struct’
  188 |     if(mod != NULL && mod->ndpi_debug_printf != NULL)                           \
      |                          ^~
reader_util.c:1775:4: note: in expansion of macro ‘NDPI_LOG’
 1775 |    NDPI_LOG(0, workflow->ndpi_struct, NDPI_LOG_DEBUG, "\n\nWARNING: IPv4 fragments are not handled by this demo (nDPI supports them)\n");
      |    ^~~~~~~~